<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonAcceptName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save the world!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>256</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
